src/main/java/com/github/havardh/javaflow/plugins/Api.java

```diff
@@ -27,4 +28,8 @@ public String getOutput() {
   public Map<String, String> getTypes() {
     return types;
   }
+
+  public Map<String, Boolean> getVerifications() {
+    return verifications;
```

Comment:
If you see my PR #2 I've rewritten all of these to be `ofNullable(...).orElse(default)`

This way we can always assume there to be proper configuration and can remove all null checks when using the values.